### PR TITLE
Added additional test case for SQL Server 

### PR DESCRIPTION
**Pull request by MrMCake**
cc: @mvbugge

[Sql: Servers](https://github.com/Azure/ResourceModules/actions/workflows/ms.sql.servers.yml)

### CODE REVIEW

File: arm/Microsoft.Sql/servers/.parameters/admin.parameters.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-sqlsrv-admin-001"
+        },
+        "administrators": {
+            "value": {
+                "azureADOnlyAuthentication": false,
```
Comment (mvbugge) [inline arm/Microsoft.Sql/servers/.parameters/admin.parameters.json:10]: "azureADOnlyAuthentication" should be `true` as no Login credential is provided.

File: arm/Microsoft.Sql/servers/.parameters/admin.parameters.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                "login": "John Doe",
```
Comment (mvbugge) [inline arm/Microsoft.Sql/servers/.parameters/admin.parameters.json]: The `login` si the object name as seen in AAD corresponding to the sid:objectId below. (Probably the name of the service principal)

**Review by mvbugge — CHANGES_REQUESTED** [review]

File: arm/Microsoft.Sql/servers/.parameters/admin.parameters.json (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.Sql/servers/.parameters/admin.parameters.json]: testing

File: arm/Microsoft.Sql/servers/deploy.bicep [hunk 3]
```diff
@@ -43,7 +43,7 @@ param firewallRules array = []
 @description('Optional. The security alert policies to create in the server')
 param securityAlertPolicies array = []
 
-@description('Optional. The Azure Active Directory (AAD) administrator authentication. Required if no `administratorLogin` & `administratorLoginPassword` is provided.')
+@description('Optional. The Azure Active Directory (AAD) administrator authentiaction. Required if no `administratorLogin` & `administratorLoginPassword` is provided.')
```
Comment (MrMCake) [inline arm/Microsoft.Sql/servers/deploy.bicep:46]:
```suggestion
@description('Optional. The Azure Active Directory (AAD) administrator authentication. Required if no `administratorLogin` & `administratorLoginPassword` is provided.')
```

**Review by ahmadabdalla — APPROVED** [review]

**Pull request merged**